tests/src/test/java/ca/bc/gov/open/jag/efiling/stepDefinitions/ViewSubmittedPackageSD.java

```diff
@@ -0,0 +1,70 @@
+package ca.bc.gov.open.jag.efiling.stepDefinitions;
+
+import ca.bc.gov.open.jag.efiling.page.AuthenticationPage;
+import ca.bc.gov.open.jag.efiling.page.PackageReviewPage;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.junit.Assert;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+
+import java.text.MessageFormat;
+import java.util.List;
+
+public class ViewSubmittedPackageSD {
+
+    @Value("${PACKAGE_REVIEW_URL:http://localhost:3000/efilinghub/packagereview}")
+    private String packageReviewUrl;
+
+    private AuthenticationPage authenticationPage;
+    private PackageReviewPage packageReviewPage;
+
+    private Logger logger = LoggerFactory.getLogger(ViewSubmittedPackageSD.class);
+
+    public ViewSubmittedPackageSD(AuthenticationPage authenticationPage, PackageReviewPage packageReviewPage) {
+        this.authenticationPage = authenticationPage;
+        this.packageReviewPage = packageReviewPage;
+    }
+
+    @Given("user is on package review page")
+    public void userIsOnPackageReviewPage() {
+
+        String packageReviewPageUrl = MessageFormat.format( "{0}/{1}", packageReviewUrl, "1");
+        logger.info("Formatted package review page url:{}", packageReviewPageUrl);
+
+        this.packageReviewPage.goTo(packageReviewPageUrl);
+        this.authenticationPage.signInWithBceid("bobross", "changeme");

```

Comment:
Could you make those environment variables, I think we will always use a single user to perform all the test, what do you think ?

Reply:
Thanks Alex, saved it for the next PR, makes sense to update in this one. fixed.